public/static/docs/command-reference/update.md

```diff
@@ -69,3 +81,41 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+## Example: Updating imported artifacts to a specified revision
+
+Let's import a data artifact from an older commit from our
+[get started example repo](https://github.com/iterative/example-get-started) at
+first:
```

Comment:
In case we keep this p:
```suggestion
Let's import a model from an old version of our
[get started example repo](https://github.com/iterative/example-get-started)
first:
```